alternate/Tenkuu Danzai Skelter Heaven (alternate).ass

```diff
@@ -26,213 +28,220 @@ Style: OPTL,Monotype Corsiva,22,&H00F8F091,&H000000FF,&H00000000,&H7F000000,-1,0
 Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text
 Comment: 0,0:00:00.00,0:00:00.00,Default,,0,0,0,,== Dialogue ============================
 Comment: 0,0:00:00.00,0:00:00.00,Default,,0,0,0,,First frame
-Dialogue: 0,0:01:07.42,0:01:11.05,Default,edit,0,0,0,,The unknown entity which appeared without warning in Tokyo airspace{edit: make these two lines complete sentences? cps is tricky}
-Dialogue: 0,0:01:13.19,0:01:15.42,Default,edit,0,0,0,,has yet to show signs of activity.{edit: make these two lines complete sentences? cps is tricky}
-Dialogue: 0,0:01:15.49,0:01:18.56,Default,,0,0,0,,Discontent towards the government is building among the citizens,
-Dialogue: 0,0:01:18.62,0:01:21.10,Default,,0,0,0,,who are confined to evacuation shelters.
-Dialogue: 0,0:01:21.16,0:01:22.98,Default,,0,0,0,,For the—
-Dialogue: 0,0:01:23.06,0:01:25.83,Default,,0,0,0,,For the first time since the war, the JSDF has been mobilized as part of a public security operation.
-Dialogue: 0,0:01:25.90,0:01:27.73,Default,,0,0,0,,How this will affect—
-Dialogue: 0,0:01:27.80,0:01:31.27,Default,,0,0,0,,...will affect international relations is uncertain.{it is believed that this will affect relations with neighbouring countries}
+Dialogue: 0,0:01:07.42,0:01:11.05,Default,edit,0,0,0,,The unknown thing which showed up in Tokyo airspace without warning{edit: make these two lines complete sentences? cps is tricky}
+Dialogue: 0,0:01:13.19,0:01:15.42,Default,edit,0,0,0,,still hasn't shown signs of moving.{edit: make these two lines complete sentences? cps is tricky}
+Dialogue: 0,0:01:15.49,0:01:18.56,Default,,0,0,0,,Anger in the citizens at the government is getting angrier,
+Dialogue: 0,0:01:18.62,0:01:21.10,Default,,0,0,0,,who is stuck in the evacuation shelter.
+Dialogue: 0,0:01:21.16,0:01:22.98,Default,,0,0,0,,For--
```

Comment:
I'm still unsure of whether the group is typographically competent enough to employ this technique. Oh well, leave it in.